deploy-artifacts.sh

```diff
@@ -0,0 +1,26 @@
+#! /bin/bash -e
+
+BRANCH=$(git rev-parse --abbrev-ref HEAD)
+
+VERSION=$BRANCH
+
+DOCKER_REPO=eventuateio
+DOCKER_COMPOSE_PREFIX=eventuateexamples_
+DOCKER_REMOTE_PREFIX=eventuate-
+
+function tagAndPush() {
```

Comment:
I think it would be simpler to have use the Gradle Docker Compose plugin to build/tag/push

Reply:
I will work on it